src/features/ModelPlan/ModelToOperations/_components/Table/index.tsx

```diff
@@ -207,119 +207,256 @@ const MTOTable = ({
-                    }
-                    onClick={e => {
-                      e.stopPropagation();
-                      setRearrangedData(
-                        moveRow(
-                          setIndexes,
-                          moveRowDirection(-1),
-                          rowType,
-                          sortedData,
-                          updateOrder,
-                          setError
-                        )
-                      );
-                    }}
-                    onKeyPress={e => {
-                      e.stopPropagation();
-                    }}
-                    className="share-export-modal__menu-item padding-y-1 padding-x-2 action-menu-item"
-                    unstyled
-                  >
-                    {t(
-                      `modelToOperationsMisc:table.menu.${rowType === 'category' ? 'moveCategoryUp' : 'moveSubCategoryUp'}`
-                    )}
-                  </Button>
-                }
-                MoveDown={
-                  <Button
-                    type="button"
-                    disabled={
-                      currentIndex === (rowLength || 0) - 1 ||
-                      currentIndex === (rowLength || 0) - 2
-                    }
-                    onClick={e => {
-                      e.stopPropagation();
-                      setRearrangedData(
-                        moveRow(
-                          setIndexes,
-                          moveRowDirection(1),
-                          rowType,
-                          sortedData,
-                          updateOrder,
-                          setError
-                        )
-                      );
-                    }}
-                    onKeyPress={e => {
-                      e.stopPropagation();
-                    }}
-                    className="share-export-modal__menu-item padding-y-1 padding-x-2 action-menu-item"
-                    unstyled
-                  >
-                    {t(
-                      `modelToOperationsMisc:table.menu.${rowType === 'category' ? 'moveCategoryDown' : 'moveSubCategoryDown'}`
-                    )}
-                  </Button>
-                }
-              />
-            ) : (
-              <>
-                {RenderCell ? (
-                  <RenderCell row={row} rowType={rowType} expanded={expanded} />
-                ) : (
-                  row[column.accessor as keyof MilestoneType]
-                )}
-              </>
-            )}
-          </td>
-        );
-      })}
-    </>
-  );
+  const RenderCells = ({
+    row,
+    rowType,
+    expanded,
+    currentIndex,
+    rowLength,
+    categoryID,
+    subCategoryID,
+    categoryIndex
+  }: {
+    row: RowType;
+    rowType: MTORowType;
+    expanded: boolean;
+    currentIndex: number;
+    rowLength: number;
+    categoryID?: string;
+    subCategoryID?: string;
+    categoryIndex?: number;
+  }) => {
```

Comment:
Converted renderCells function to a functional component to aid in no longer needing to remember the function parameter order when calling the function and not needing to write out `undefined` for optional parameters.

i.e. 
```js
{renderCells(row, rowType, false, 0, 0, undefined, undefined, undefined)}
``` 
vs
```js
<RenderCells 
  row={row}
  rowType={rowType}
  expanded={false}
  currentIndex={0}
  rowLength={0}
// categoryID, subCategoryID, categoryIndex omitted
/>
``` 

It was easier for me to keep things in my head when developing 😂